rai/node/bootstrap.cpp

```diff
@@ -1115,21 +1191,8 @@ void rai::bootstrap_attempt::stop ()
 
 void rai::bootstrap_attempt::add_pull (rai::pull_info const & pull)
 {
-	static rai::account landing ("059F68AAB29DE0D3A27443625C7EA9CDDB6517A8B76FE37727EF6A4D76832AD5");
```

Comment:
This doesn't help much at all, I tried doubling down and prioritizing a larger set of accounts to try to have fewer unchecked blocks, but it's not worth it.